app/wizards/schools/register_mentor_wizard/wizard.rb

```diff
@@ -66,6 +71,12 @@ def allowed_steps
             return steps unless mentor.email
             return steps + %i[change_email_address cant_use_changed_email cant_use_email] if mentor.cant_use_email?
 
+            steps << :mentoring_at_new_school_only
+            steps << :started_on
+            steps << :previous_training_period_details
+            steps << :programme_choices
+            steps << :lead_provider
```

Comment:
I think this is incomplete/or breaking the purpose of this method to return the allowed steps a user can go from a given step.